src/components/searchResults/searchResults.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { hot } from "react-hot-loader";
+import "./searchResults.scss";
+import MovieList from "../movieList/movieList";
+
+
+class SearchResults extends React.PureComponent {
+    constructor (props) {
+        super(props);
+        this.state = {}; // is necessary to show result/no result part
+    }
+
+    render () {
+        return (
+            <div className="searchResults__wrapper">
+                <div className="container">
+                    <div className="searchResults">
+                        <h2 className="searchResults__title">No films found</h2>
+                    </div>
+                    <div className="searchResults">
```

Comment:
to get rid of two `searchResults` divs you can display `MovieList` or `NoResults` depending on the movies amount passed to this component

Reply:
Will be done in the scope of Redux task (added TODO)